[Translate ENG-KO] updates/2016/05/Houdini.md

jihyerish opened the pull request:
I've translated the updates about houdini APIs.

jihyerish commented:
Hi, @cwdoh. 
I've found you in the list of WebFundamentals moderators.
This is the first time for me to contribute WebFundamentals. 
So I'm wondering how to proceed with this PR, so I've contacted you.
Could you give me some guideline and review this PR?

Thanks in advance! : )

ragingwind commented:
Great job! @jihyerish I guess you can try the translation with this https://developers.google.com/web/resources/translations But please considering that it is in the experimental stage yet.

jihyerish commented:
@ragingwind Thanks for the guideline! Using gitlocalize seems really convenient!
Then do I need to close this PR and start on the gitlocalize?

jenix21 commented:
@jihyerish @ragingwind 
This pr is on progress?

jihyerish commented:
@jenix21 I guess this PR is waiting for the review.

jenix21 commented:
Thanks @jihyerish 
Reviews at https://gitlocalize.com/repo/107/ko/review appear to be not that active.

jihyerish commented:
I've requested the review on https://gitlocalize.com/repo/107/ko/review/8482.

ragingwind commented:
@jihyerish I will let reviewers to take on this. please understand that it's not so very activated yet.

jihyerish commented:
@ragingwind Sure. :) Thank you for your support!

cwdoh commented on src/content/ko/updates/2016/05/houdini.md:
```diff
@@ -0,0 +1,265 @@
+*표준 초안*들을 작성하고 있습니다. 프로젝트의 구성원들은 표준안에 대한 몇 가지 높은 수준의 목표를 세워 표준 초안을 작성하였으며 이를 기반으로
+하위 수준 사양의 표준 문서들이 개발되었습니다. 결국, “Houdini”라하면 이 표준 문서들의 내용을 의미합니다.
+표준안 개발 작업이 진행되는 동안 [Houdini 표준안 초안][Houdini Drafts]들은 미완성 단계이며 일부 초안은 다소 가안입니다.
+이것이 Houdini를 개발하는 초기 단계입니다.
```
문장을 좀 의역해보는게 어떨까요?
> 지금까지 Houdini의 초기 개발 단계가 어떻게 진행되고 있는지 설명해드렸습니다.

cwdoh commented on src/content/ko/updates/2016/05/houdini.md:
```diff
@@ -0,0 +1,265 @@
+CSS 엔진의 일부분에 웹 개발자들이 접근할 수 있도록 협업하고 있습니다. 이 프로젝트는 공식 W3C 표준으로 채택되기 위한
+*표준 초안*들을 작성하고 있습니다. 프로젝트의 구성원들은 표준안에 대한 몇 가지 높은 수준의 목표를 세워 표준 초안을 작성하였으며 이를 기반으로
+하위 수준 사양의 표준 문서들이 개발되었습니다. 결국, “Houdini”라하면 이 표준 문서들의 내용을 의미합니다.
+표준안 개발 작업이 진행되는 동안 [Houdini 표준안 초안][Houdini Drafts]들은 미완성 단계이며 일부 초안은 다소 가안입니다.
```
제안
> 표준안 초안 -> 초안
> ... 완료되지 않았으며, 초안 중 일부는 작업 예정 사항입니다.

cwdoh commented on src/content/ko/updates/2016/05/houdini.md:
```diff
@@ -0,0 +1,265 @@
+
+이는 곧 web workers가 Houdini가 하려고 했던 것을 하기에는 무리라는 의미입니다. 따라서 worklets이 만들어졌습니다.
+Worklets은 ES2015의 class들을 써서 method 집합들, worklet의 미리 정의된 타입인 signature들을 정의합니다.
+그렇기 때문에 Worklet은 보다 경량화가 되었으며 수명 주기가 짧습니다.
```
보통 [생명 주기](http://terms.tta.or.kr/mobile/dictionaryView.do?word_seq=047526-1)를 좀 더 많이 사용합니다.

cwdoh commented on src/content/ko/updates/2016/05/houdini.md:
```diff
@@ -0,0 +1,265 @@
+새로 디자인되었습니다. 해당 API에 대한 더 자세한 사항은
+[여기](https://dassur.ma/things/animworklet/)에서 찾아보실 수 있습니다.
+
+Compositor Worklet이 WICG로 옮겨졌고 계속 내용이 바뀌고 있지만, 저에게는 가장 흥미로운 표준안입니다. 아시다시피 일부 작업은 CSS 엔진에 의해 그래픽 카드로 아웃소싱되며 일반적으로 그래픽 카드와 장치 모두에 의존합니다. 브라우저는 일반적으로 DOM 트리를 사용하고 특정 기준에 따라 일부 branch 혹은 하위 트리에 자체 [레이어][HTML5Rocks layers]를 공개합니다.
```
`저에게는`는 번역 시 제외하시는게 더 자연스러울 것 같습니다. :)

cwdoh commented on src/content/ko/updates/2016/05/houdini.md:
```diff
@@ -0,0 +1,265 @@
+[여기](https://dassur.ma/things/animworklet/)에서 찾아보실 수 있습니다.
+
+Compositor Worklet이 WICG로 옮겨졌고 계속 내용이 바뀌고 있지만, 저에게는 가장 흥미로운 표준안입니다. 아시다시피 일부 작업은 CSS 엔진에 의해 그래픽 카드로 아웃소싱되며 일반적으로 그래픽 카드와 장치 모두에 의존합니다. 브라우저는 일반적으로 DOM 트리를 사용하고 특정 기준에 따라 일부 branch 혹은 하위 트리에 자체 [레이어][HTML5Rocks layers]를 공개합니다.
+이러한 하위 트리는 스스로 해당 레이어에 그립니다. (향후에는 paint worklet을 사용하여 그릴 수도 있겠죠!) 마지막 단계로, 페인트 과정이 완료된 모든 개별 레이어는 z-index, 3D Transform등을 고려하여 서로 중첩이 되어 화면에 보이는 최종 이미지를 완성합니다. 이 프로세스를 “컴포지팅 (Compositing)”이라고 하며 컴포지터(Compositor)가 실행합니다. 이 프로세스의 장점은 페이지가 조금 스크롤될 때 *모든* 요소를 다시 그릴 필요가 없다는 것입니다. 대신, 이전 프레임에서 레이어를 재사용하고 변경된 스크롤 위치로 컴포지터를 다시 실행할 수 있습니다. 이로 인하여 빠른 처리가 가능합니다. 이는 60fps를 맞출수 있게 해줍니다. 이런 성능 개선은 [Paul Lewis ...
```
제안
> 해줍니다. -> 해줄 것입니다.

cwdoh commented on src/content/ko/updates/2016/05/houdini.md:
```diff
@@ -0,0 +1,265 @@
+### Layout worklet
+([spec][Layout Worklet spec])
+
+참고: 첫 공식 표준안이 제안되었음 표준안의 구현도 상당히 진행되었습니다.
```
`... 제안되었으며, 표준안의 구현도...`와 같이 원문을 조금 바꾸셔도 좋을 듯 합니다.

cwdoh commented on src/content/ko/updates/2016/05/houdini.md:
```diff
@@ -0,0 +1,265 @@
+물론, CSS `flex-box` Layout을 모두 자바스크립트로 구현하는 것은 동일한 기능을 native에서
+실행하는 것보다 느리겠지만 성능 상의 이득을 얻을 수 있는 지름길을 택하는 시나리오를 생각해볼 수 있습니다.
+ Windows 10 또는  [Masonry] 스타일의 레이아웃으로 구성된 웹 페이지를 상상해보세요. absolute/fixed positioning은 사용되지 않으며,
+`z-index` 나 요소가 겹치거나 어떤 종류의 border 혹은 overflow도 없습니다. 재 레이아웃 시 이러한 모든 검사를 건너뛸 수 있으면 성능이 향상될 수 있습니다.
```
`재 레이아웃`에 `(re-layout)`을 추가해주시면 더 좋을 것 같습니다. :)

cwdoh commented on src/content/ko/updates/2016/05/houdini.md:
```diff
@@ -0,0 +1,265 @@
+### Properties and values
+([spec][Properties and Values spec])
+
+참고: 표준안은 상당히 완성되었습니다. 하지만 아직 사용 가능한 구현은 없습니다.
```
제안
> ... 되었습니다만, 아직 ...

cwdoh commented on src/content/ko/updates/2016/05/houdini.md:
```diff
@@ -0,0 +1,265 @@
+참고: 표준안은 상당히 완성되었습니다. 하지만 아직 사용 가능한 구현은 없습니다.
+
+[CSS Custom Properties] (비공식적으로는 “CSS Variables”로도 불립니다.) 를 아시나요?
+이것은 타입이 있는 변수입니다! 지금까지, 변수들은 string 값들 만을 가지거나
```
`들 만`으로 띄어쓰기가 되어 있습니다.

ragingwind commented:
@cwdoh @jihyerish already using gitlocalize, https://gitlocalize.com/repo/107/ko/review/8482

jihyerish commented:
> @jihyerish 개인적인 사정으로 한동안 WebFundamentals 프로젝트에 신경을 쓰지 못했습니다. 늦었지만 PR에 대한 리뷰를 추가하였으니, 한번 살펴봐주시면 고맙겠습니다. 번역하시느라 정말 수고하셨습니다. :)
> 
> Sorry for my late review on your PR, and also just wanna say Thank you for great work. I addressed some nits on the translation, but there's no error to understand meaning.

우와 정말 감사드립니다!! 리뷰해주신 내용 여기에서 바로 수정하면 될까요?
혹은 gitlocalize로 고치면 될까요?
WebFundamentals 번역 처음해보아서 절차가 어떻게 되는지 궁금합니다.

cwdoh commented:
@jihyerish 어디에 작업하셔도 괜찮습니다.

@ragingwind I know, but I'm confused to review PR with GitLocalize. I think
I've been away from this project for so long.

2019년 4월 1일 (월) 오전 10:37, Jihye Hong <notifications@github.com>님이 작성:

> @jihyerish <https://github.com/jihyerish> 개인적인 사정으로 한동안 WebFundamentals
> 프로젝트에 신경을 쓰지 못했습니다. 늦었지만 PR에 대한 리뷰를 추가하였으니, 한번 살펴봐주시면 고맙겠습니다. 번역하시느라 정말
> 수고하셨습니다. :)
>
> Sorry for my late review on your PR, and also just wanna say Thank you for
> great work. I addressed some nits on the translation, but there's no error
> to understand meaning.
>
> 우와 정말 감사드립니다!! 리뷰해주신 내용 여기에서 바로 수정하면 될까요?
> 혹은 gitlocalize로 고치면 될까요?
> WebFundamentals 번역 처음해보아서 절차가 어떻게 되는지 궁금합니다.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/google/WebFundamentals/pull/6963#issuecomment-478407571>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACBcD_vSEIjYcCSCYPxy1wa5lKzl9XSlks5vcWLygaJpZM4ZK6ax>
> .
>

ragingwind commented:
@cwdoh I thought that gitlocalize is the platform we should use. @agektmr What are your thought on this?

agektmr commented:
@ragingwind Ideally using GitLocalize would help. But it's flexible enough to do it otherwise. We are working closely with GitLocalize team to improve your translation experience. Stay tuned. @cwdoh

cwdoh commented:
LGTM. I'll be back soon with gitlocalize :)

agektmr commented:
@cwdoh @ragingwind There still remains some comments to resolve. Do you think we are good to go?

ragingwind commented:
@agektmr I thought that some of unresolves are minor issue and we can keep doing it on gitlocalize.

agektmr commented:
@ragingwind ok, then I'll wait for LGTM from @cwdoh .

jihyerish commented:
Oh, I just see the comments above. Thank you for taking care of this PR!
I'll continue working this via gitlocalize!

agektmr commented:
@jihyerish @ragingwind @cwdoh Thank you for your contribution, all!
